Doc/c-api/long.rst

```diff
@@ -494,6 +494,21 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.13
 
 
+.. c:function:: Py_ssize_t PyLong_GetNumBits(PyObject *obj)
+
+   Get the number of bits of an integer.
+
+   * Return the number of bits on success: greater than or equal to zero.
+   * Set an exception and return ``-1`` on error.
+   * Set an :exc:`OverflowError` exception, and return ``-1`` if the number
+     of bits doesn't fit into :c:type:`Py_ssize_t`.
+
+   Calling the :py:meth:`bit_length` method should be preferred to support integers
```

Comment:
```suggestion
   Calling the :py:meth:`!bit_length` method should be preferred to support integers
```

Reply:
I added a link to the Python method instead.